modules/rosa-hcp-creating-account-wide-sts-roles-and-policies.adoc

```diff
@@ -15,10 +15,17 @@ Before using the {product-title} (ROSA) CLI, `rosa`, to create {hcp-title-first}
 
 .Procedure
 
+. Set your account role prefix:
++
+[source,terminal]
+----
+$ ACCOUNT_ROLES_PREFIX="${PREFIX:-ManagedOpenShift}"
```

Comment:
I think {PREFIX:-ManagedOpenShift} should be {PREFIX}

Reply:
This was just intended to give users a default value in the environment variable. Since the `rosa` CLI defaults to `ManagedOpenShift`, it made sense to do the same in case a `${PREFIX}` was not previously set.

Reply:
Spoke to QE and agreed to remove ":-ManagedOpenShift" and replace with {OPERATOR_ROLES_PREFIX} to be consistent with "ACCOUNT_ROLES_PREFIX"

Reply:
I'm not seeing this reflected at all in the document anymore